test/unittest/StatisticsBackend/CMakeLists.txt

```
@@ -168,6 +168,11 @@ foreach(test_name ${USER_LISTENERS_DEATH_TEST_LIST})
         COMMAND calling_user_listeners_tests
         --gtest_filter=calling_user_listeners_DeathTest.${test_name}:*/calling_user_listeners_DeathTest.${test_name}/*)
 
+    set_tests_properties(calling_user_listeners_DeathTest.${test_name}
```

Restore the comment, it was a good hint
```suggestion
    // Note the Death Tests (like this one) only work on Debug config because assert() will only
    // terminate the process in that case
    // TODO(Miguel C.): Check why this tests fails on termination in Windows
    set_tests_properties(calling_user_listeners_DeathTest.${test_name}
```